weave/tests/test_evaluations.py

```diff
@@ -144,3 +152,366 @@ def is_op_ref_with_name(val: Any, name: str):
     model_obj = objs.objs[0]
     expected_predict_ref = model_obj.val["predict"]
     assert is_op_ref_with_name(expected_predict_ref, "MyModel.predict")
+
+
+@weave.op
+def gpt_mocker(question: str):
+    return {
+        "response": str(question),
```

Comment:
the type annotation suggests it is already a str

Reply:
👍 in next commit